### ISSUE TITLE
Deprecation warning if missing a command

### ISSUE THREAD

**Issue by jillbert** [rails/importmap-rails#35]
```
% ./bin/importmap plyr
Could not find command "plyr".
Deprecation warning: Thor exit with status 0 on errors. To keep this behavior, you must define `exit_on_failure?` in `Importmap::Commands`
You can silence deprecations warning by setting the environment variable THOR_SILENCE_DEPRECATION.
```

The correct command would be `./bin/importmap pin plyr`, but just noting the deprecation warning :)

**Comment by dhh** [comment]
Good catch! It's a bit of a weird notice. Like, what are we actually supposed to do here? If you want to look into it and do a patch with whatever is right 🙏

**Comment by dhh** [comment]
Fixed via 3731428
